test/transform/test_graph_to_simplicial_complex.py

```diff
@@ -52,6 +53,100 @@ def test_graph_2_clique_complex(self):
 
         return
 
+    def test_weighted_graph_to_Vietoris_Rips_complex(self):
+        """Test weighted_graph_to_Vietoris_Rips_complex"""
+
+        def generate_weighted_graph_for_Vietoris_Rips():
+            """Creates a weighted graph in networkx used to test the lift from
+            undirected weighted graphs to Vietoris-Rips simplicial complexes.
+
+            Returns
+            -------
+            networkx graph
+                A undirected weighted graph
+            """
+            G = nx.Graph()
+            # We add a 3-clique to the graph with pairwise weights 1.0
+            G.add_edge(0, 1, weight=1.0)
+            G.add_edge(0, 2, weight=1.0)
+            G.add_edge(0, 3, weight=1.0)
+            G.add_edge(1, 2, weight=1.0)
+            G.add_edge(1, 3, weight=1.0)
+            G.add_edge(2, 3, weight=1.0)
+            # We add a 2-clique to the graph with pairwise weights 2.0
+            G.add_edge(4, 5, weight=2.0)
+            G.add_edge(4, 6, weight=2.0)
+            G.add_edge(5, 6, weight=2.0)
+            # We add an edge between the two cliques with weight 3.0
+            G.add_edge(3, 4, weight=3.0)
+            # We add a new vertex 7 to the graph that forms a 3 clique with the previous
+            # vertices 4, 5, and 6 if we take r > 4.0
+            G.add_edge(4, 7, weight=4.0)
+            G.add_edge(5, 7, weight=4.0)
+            G.add_edge(6, 7, weight=4.0)
+            return G
+
+        def generate_expected_simplices_for_Vietoris_Rips_complex(r):
+            """Generates a pair of lists of tuples containing the expected and non-expected simplices of the
```

Comment:
Docstring need to respect formatting + convention given in the contributing guidelines on the website. E.g. Generates --> Generate + other minor issues.

Reply:
All the comments applied! Thank you @ninamiolane!